### PR TITLE
4746/bug/fix svgo config

### LINKED ISSUE #4746: Fix svgo and minify SVG assets
when running `npm run svg-min` i get:

```
/openlibrary/.svgo.yml:2
  - removeXMLProcInst: false
                     ^

SyntaxError: Unexpected token ':'
    at wrapSafe (internal/modules/cjs/loader.js:915:16)
    at Module._compile (internal/modules/cjs/loader.js:963:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1027:10)
    at Module.load (internal/modules/cjs/loader.js:863:32)
    at Function.Module._load (internal/modules/cjs/loader.js:708:14)
    at Module.require (internal/modules/cjs/loader.js:887:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at importConfig (/openlibrary/node_modules/svgo/lib/svgo-node.js:12:18)
    at loadConfig (/openlibrary/node_modules/svgo/lib/svgo-node.js:33:20)
    at Command.action (/openlibrary/node_modules/svgo/lib/svgo/coa.js:105:32)
```

This should be fixed and we should minify all existing SVGs in the repo.

### Steps to Reproduce

1. Go to ...
2. Do ...

* Actual: 
* Expected: 

### Details

- **Logged in (Y/N)?** 
- **Browser type/version?** 
- **Operating system?** 
- **Environment (prod/dev/local)?** prod

### Stakeholders
@jdlrobson

### PR DESCRIPTION
**Pull request by jonbos**
Closes #4746 

This PR **fixes** an issue with the `svg-min` script in package.json. The script  was failing with an error related to the syntax of the yml file. The svgo [README](https://github.com/svg/svgo#configuration) illustrates use of a config module - this PR creates that module and replaces the .svgo.yml file.

### Technical

The svgo README specifies 

> The default list is fully overridden if the plugins field is specified. To extend the default list use the extendDefaultPlugins utility:

Thus the new config module specifies only **extension** of default plugins (this matches all plugins being set to `false` in the .svgo.yml)

### Testing

Run `svg-min` to successfully optimize the svg file specified in the script
### Stakeholders

@jdlrobson

### CODE REVIEW

File: svgo.config.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/* eslint-env node, es6 */
```
Comment (cclauss) [inline svgo.config.js:1]: Do we _need_ to put this file at the root of the repo?  The root dir is already quite cluttered and this filename does not start with a . so it is not hidden in directory listings.  Could this file live in conf / config or some other directory or _must_ it live at the root?

Comment (jonbos) [reply svgo.config.js:1]: Agreed on all points @cclauss - there is nothing preventing that script residing in /conf/config. I will update the PR. Thank you.

Comment (cclauss) [reply svgo.config.js]: /config is an alias that points to /conf so put the file in /conf and then look for it in either directory.

File: conf/config/svgo.config.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+/* eslint-env node, es6 */
```
Comment (cclauss) [inline conf/config/svgo.config.js:1]: Sorry...  I was unclear.  The path to this file should be `conf/svgo.config.js`

File: package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "build-assets": "make js && make css && make components",
     "build-assets:css": "make css",
     "build-assets:js": "make js",
-    "svg-min": "svgo --config .svgo.yml static/images/**/*.svg static/images/*.svg",
+    "svg-min": "svgo --config conf/config/svgo.config.js static/images/**/*.svg static/images/*.svg",
```
Comment (cclauss) [inline package.json:12]:
```suggestion
    "svg-min": "svgo --config conf/svgo.config.js static/images/**/*.svg static/images/*.svg",
```

**Review by cclauss — APPROVED** [review]
RSLGTM

File: package.json (hunk 3 above)
Comment (jonbos) [reply package.json]: @cclauss I appreciate your patience - thank you for walking me through that.

**Comment by jdlrobson** [comment]
Sorry for the delay here. This is awesome and does exactly what it says it does, so thank you!

 @jonbos would you be interested in a follow-up to run the command and sending a pull request with the minified assets? Unfortunately, the minification doesn't happen automatically as part of our build step yet.

	modified:   static/images/facebook.svg
	modified:   static/images/openlibrary-icon.svg
	modified:   static/images/pinterest.svg
	modified:   static/images/twitter.svg

**Pull request merged**